demos/local-only-todolist/README.md

```diff
@@ -0,0 +1,106 @@
+# PowerSync + Supabase Flutter Todo List App: Local-Only to Sync Mode Demo
+
+This demo app is an extension of the [Supabase Todo List App](../supabase-todolist/README.md) and demonstrates using the PowerSync Flutter client SDK to persist data locally without syncing, for users to use the app without having to register or sign in. It then demonstrates syncing this data to Supabase once the user registers at a later stage.
```

Comment:
This demo app is an extension of the [Supabase Todo List App](../supabase-todolist/README.md) and demonstrates using the PowerSync Flutter client SDK to persist data locally without syncing. This lets users use the app without having to register or sign in.